.github/workflows/build-test-measure.yml

```diff
@@ -425,8 +434,7 @@ jobs:
         with:
           php-version: ${{ matrix.php }}
           extensions: curl, date, dom, gd, iconv, json, libxml, mysql, spl
-          coverage: ${{ matrix.coverage && 'pcov' || 'none' }}
-          ini-values: pcov.directory=.
+          coverage: ${{ matrix.coverage && 'xdebug' || 'none' }}
```

Comment:
Should we continue using `pcov` for coverage in the CI? I've looked at the problem that it uses xdebug in any case, but I can debug it in another PR. Thoughts?

Reply:
I'll defer to Weston, but if it works and is faster, sure.

But since this never really worked in the first place, definitely warrants its own issue/PR.

Reply:
PCOV is faster but lacks path coverage. A difference in reporting can be seen [here](https://github.com/krakjoe/pcov#differences-in-reporting).

Reply:
PCOV was introduced in https://github.com/ampproject/amp-wp/pull/5201

I just pinged you both the thread where this was discussed.